### DISCUSSION TITLE
CSS styles and images not rendered by server in DEV/PROD mode.

### DISCUSSION THREAD

**Discussion by subaruMax** [vercel/next.js#28259] [Help] [answered]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

14.17.3

### What browser are you using?

Chrome

### What operating system are you using?

Windows 10

### How are you deploying your application?

next start

### Describe the Bug

Hi, i'v got issue with styles and images not rendered on server side in nextjs typescript app, on client everything works fine. I am using sass modules, and normalize.css in my project. Even if I create templete next.js app i will got the same issue, in response page will be displayed without any styles and images. I tried both development and production mode. 
![1](https://user-images.githubusercontent.com/64718566/129888883-62ec4bf5-b9b7-45fe-961b-29cc2c0d6efa.PNG)
![2](https://user-images.githubusercontent.com/64718566/129888885-bcb3371d-4c15-43a0-a1e5-a48dbd2bc558.PNG)

### Expected Behavior

Expecting that styles and images rendering will be done on server side. For example like https://nextjs.org/ see screenshot.
![3](https://user-images.githubusercontent.com/64718566/129889018-e8cfb666-69c9-4c2e-a093-bb7446db3c9f.PNG)

### To Reproduce

1. 
`npx create-next-app --ts`
# or
`yarn create next-app --typescript`

2.
```
npm run build
npm run start
```
3. Open DevTools -> Network tab -> click on file localhost(type: document) -> click on Preview tab

**Comment by timneutkens** [comment, marked as answer]
This is expected, the chrome devtools only render the exact response without `<link>` or `<script>` in the network tab and using the network tab to verify if styles are loaded will not work. The reason that you're seeing the styles for nextjs.org is that it is using styled-jsx which inlines rendered component styles into the html response. There's no difference between that or your example as both still load styles. Only difference is that styled-jsx causes inline styles to be used.

In general I'm very curious how you came to think that the styles should be displayed there as I've been bringing this particular misconception up to the Chrome team a few times now and I'd like to provide more context around it 🙏
